### PR TITLE
Feature/gallery

### PR DESCRIPTION
**Pull request by lpyvnytska**

### CODE REVIEW

File: server/services/tokenService.js [hunk 1]
```diff
@@ -35,7 +35,12 @@ const generateRefreshToken = user => {
 };
 
 //Replace refreshToken in DB token
```
Comment (yushevchuk) [inline server/services/tokenService.js]: Remove comment, please.

File: server/services/tokenService.js [hunk 2]
```diff
@@ -35,7 +35,12 @@ const generateRefreshToken = user => {
...
-const replaceDbRefreshToken = async (tokenId, userId, expiredAt, oldRefreshTokenId) => {
+const replaceDbRefreshToken = async (
+  tokenId,
+  userId,
+  expiredAt,
+  oldRefreshTokenId
+) => {
   //console.log('token ' + tokenId + 'oldRefreshTokenId ' + oldRefreshTokenId);
```
Comment (yushevchuk) [inline server/services/tokenService.js]: Remove comment, please.

File: server/services/tokenService.js (hunk 1 above)
Comment (lpyvnytska) [reply server/services/tokenService.js]: done

File: server/services/tokenService.js (hunk 2 above)
Comment (lpyvnytska) [reply server/services/tokenService.js]: done

**Pull request merged**